index.html

```diff
@@ -0,0 +1,165 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+<!-- HTML File for Davins Portfolio-->
+  <title>Davin's Portfolio</title>
+  <link href="normalize.css" type="text/css" rel="stylesheet">
+  <link href="style.css" type="text/css" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.7/css/all.css">
+</head>
+
+<body>
+	<div class="mainContainerGrid">
+<!-- Header & Navigation -->
+		<header class="colorHeaderFooter fontCursive">
```

Comment:
these classes are not semantic, you should decrobe what the thing IS not what it looks like. its not a header and a footer, its certainly not a color

Reply:
most of your classes are not semantic throughout